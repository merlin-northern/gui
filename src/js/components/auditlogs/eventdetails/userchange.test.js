import React from 'react';
import { render } from '@testing-library/react';
import { defaultState, undefineds } from '../../../../../tests/mockData';
import UserChange from './userchange';

describe('UserChange Component', () => {
  it('renders correctly', async () => {
    const { baseElement } = render(<UserChange item={defaultState.organization.events[0]} />);

    const view = baseElement.firstChild.firstChild;
    expect(view).toMatchSnapshot();
    expect(view).toEqual(expect.not.stringMatching(undefineds));
  });
});
