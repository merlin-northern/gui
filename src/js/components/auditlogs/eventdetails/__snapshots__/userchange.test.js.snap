// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserChange Component renders correctly 1`] = `
<div
  class="code flexbox column"
>
  <span
    class=""
  >
    change1
  </span>
</div>
`;
